### ISSUE TITLE
20 properties per account limit

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gzog/analytics-issues#506]
```
There is an upper bound of 20 properties per account.  This is according to 
https://developers.google.com/analytics/devguides/config/mgmt/v3/mgmtWebproperty
Guide. There is no such limit for Analytics UI users apparently.

This is the error that I am getting if I try to add more than 20 web properties 
to my account:

(400) Error creating this entity. You have reached the maximum allowed entities 
of this type.

I tried to get this limit lifted by filling out the Analytics API quota request 
form but to no avail.


```

Original issue reported on code.google.com by `a...@totalwebservices.net` on 4 Oct 2014 at 12:51

**Comment by GoogleCodeExporter** [comment]
Original comment by `pfrise...@google.com` on 10 Oct 2014 at 6:28
- Changed state: **WontFix**

**Comment by GoogleCodeExporter** [comment]
```
Status: WontFix ???

No comment?  Whyyyyyy???????????.........

░░░░░░░░░░░░██████▀▀▀█████��
�█░░░░░░░░░░░░
░░░░░░░░▄▄▄▄█▀▀░░░░░▄███▀▀��
�█░░░░░░░░░░░░
░░░░░░░█████▄░░▀▄▄░░███▀▀█��
�██░░░░░░░░░░░
░░░░░░░████▀██▄░░░░░██░░░▀��
�█████░░░░░░░░
░░░░░░░███░░░░▀▄▄▄▄░██░░░░��
�░░░▀█░░░░░░░░
░░░░░░███▀░░░░░████▀▀▀█▄░░��
�░░░░██░░░░░░░
░░░░░██░░░░░░░░█░░▄▄▄▄▄██▄��
�░░░░▄█░░░░░░░
░░░░░██░░░░░░░▄▀▄▀░█░▄█▄██��
�▀▄▄▀▀█▄░░░░░░
░░░░░███░░░░░▄▀█░█▄███████��
�▄░░░░██░░░░░░
░░░░░██░▀▀▀▀▀░▄███████████��
�██░░░██░░░░░░
░░░░░██▄░░░░░░██████▀▀▀░░▀��
�███░░██░░░░░░
░░░░░░██░░░░▄▀████▀░░░░░░▄��
�▀███░██░░░░░░
░░░░░░░██░░░█████░░▄░░░░▄█��
�░▀██▄██░░░░░░
░░░░░░░░██░░░███░░░░▀▀▄▄▄▄��
�▄░█▀███░░░░░░
░░░░░░░░░███▄███▄▄▄▄███▄▄█��
�▄█████░░░░░░░

```

Original comment by `a...@totalwebservices.net` on 10 Oct 2014 at 6:50

**Comment by GoogleCodeExporter** [comment]
```
TL;DR - this is not a defect and everything is working as documented.

1) This is documented as you linked to which means the behavior does not 
deviate from this so it is not a defect or bug. 2) Quota increase request form 
does not list properties as a limit that can be increased. 3) UI and API will 
likely not have this difference in the future.
```

Original comment by `pfrise...@google.com` on 10 Oct 2014 at 7:19

**Comment by GoogleCodeExporter** [comment]
```
Thank you for the response.

I do in fact think this is a defect, but in the design of the API.  The API 
needs to allow this if users are allowed to do this manually using the UI.

When are you planning to lift this limit?
Can this limit be lifted for my account (54768316) manually at all?

```

Original comment by `a...@totalwebservices.net` on 10 Oct 2014 at 8:48

**Comment by GoogleCodeExporter** [comment]
```
Communicating with Google devs is like talking to a wall.  Nothing of value 
will be gained and a bunch of time will be lost.

I really need to be able to create more than 20 properties in my Analytics 
account or I will not be able to make use of the API.

```

Original comment by `a...@totalwebservices.net` on 13 Oct 2014 at 10:14

**Comment by GoogleCodeExporter** [comment]
```
The is no longer a separate limit between the API and UI. The documentation has 
been updated. Please update this if you receive an Error.
```

Original comment by `mcoh...@google.com` on 20 Apr 2015 at 11:16
- Changed state: **Fixed**
